models/org.go

```diff
@@ -563,18 +563,22 @@ func (org *User) getUserTeams(e Engine, userID int64, cols ...string) ([]*Team,
 		Find(&teams)
 }
 
+func (org *User) getOrgTeamIDs(e Engine, userID int64) ([]int64, error) {
+	teamIDs := make([]int64, 0, org.NumTeams)
+	return teamIDs, e.
+		Table("team").
+		Cols("team.id").
+		Where("`team_user`.org_id = ?", org.ID).
+		Join("INNER", "team_user", "`team_user`.team_id = team.id").
+		Join("INNER", "user", "`user`.id=team_user.uid").
+		And("`team_user`.uid = ?", userID).
+		Asc("`user`.name").
```

Comment:
Why is this `ORDER BY` necessary? We are only looking for one user (i.e. the user with ID `userID`)

Plus, if we get rid of the `ORDER BY`, we won't need to join with the `user` table :smile: